### DISCUSSION TITLE
Project Status

### DISCUSSION THREAD

**Discussion by geolffreym** [latchset/jwcrypto#321] [Q&A]
Hello guys, i am implementing some features using jwcrypto, i need to know the state of the project?

Thank you

**Comment by simo5** [comment]
The project exists and implements all we think is needed to handle JWT/JWE/JWS.
If you have a specific feature request feel free to open an issue. For general questions, please use the discussions.

**Reply by geolffreym** [reply]
Awesome! Good to know
